soroban-token-spec/Cargo.toml

```diff
@@ -29,3 +29,4 @@ rand = { version = "0.7.3", optional = true }
 soroban-auth = { workspace = true, features = ["testutils"] }
 ed25519-dalek = { version = "1.0.1" }
 rand = { version = "0.7.3" }
+stellar-xdr = { workspace = true }
```

Comment:
Can you use the `soroban_sdk::xdr` instead of `stellar-xdr`? We export the xdr in the SDK, and that helps to keep the dependency tree easier to navigate.

Reply:
Updated